<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Images/logo bank.png</normaloff>:/Images/logo bank.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#SignUp{		
background-image: url(&quot;:/Images/sign up page.png&quot;);
        background-repeat: no-repeat;
        background-attachment: fixed;
        background-size: cover;
        background-position: center;
}
</string>
  </property>
  <widget class="QLineEdit" name="lnEdtName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lnEdtName{
    background-color:  #ffb4b6; /* رنگ پس‌زمینه مشکی */
    color: #3f0002; /* رنگ متن سفید */
	opacity: 0.5;
    border: 2px solid #ffb4b6; /* حاشیه با رنگ قرمز */
    border-radius: 12px; /* گرد کردن حاشیه */
	border-style: inset;
    padding: 3px 10px; /* فاصله داخلی دکمه */
    font-size: 20px; /* اندازه‌ی متن */
	font-family:'Cascadia Mono' ;
	font-weight: bold;
    text-align: center; /* مرکز‌چین کردن متن */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblName {
    color: white; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
    font-size: 27px; /* اندازه‌ی متن (مثلاً 20 پیکسل) */
    border-bottom: 2px solid red; /* حاشیه زیر متن با رنگ قرمز */
    box-shadow: 20 20 30px red; /* افکت shadow با رنگ قرمز */
    padding: 10px; /* فاصله داخلی دکمه */
}

</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFamily">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblFamily {
    color: white; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
    font-size: 27px; /* اندازه‌ی متن (مثلاً 20 پیکسل) */
    border-bottom: 2px solid red; /* حاشیه زیر متن با رنگ قرمز */
    box-shadow: 20 20 30px red; /* افکت shadow با رنگ قرمز */
    padding: 10px; /* فاصله داخلی دکمه */
}

</string>
   </property>
   <property name="text">
    <string>Family:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdtFamily">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lnEdtFamily{
    background-color:  #ffb4b6; /* رنگ پس‌زمینه مشکی */
    color: #3f0002; /* رنگ متن سفید */
	opacity: 0.5;
    border: 2px solid #ffb4b6; /* حاشیه با رنگ قرمز */
    border-radius: 12px; /* گرد کردن حاشیه */
	border-style: inset;
    padding: 3px 10px; /* فاصله داخلی دکمه */
    font-size: 20px; /* اندازه‌ی متن */
	font-family:'Cascadia Mono' ;
	font-weight: bold;
    text-align: center; /* مرکز‌چین کردن متن */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAge">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>123</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblAge {
    color: white; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
    font-size: 27px; /* اندازه‌ی متن (مثلاً 20 پیکسل) */
    border-bottom: 2px solid red; /* حاشیه زیر متن با رنگ قرمز */
    box-shadow: 20 20 30px red; /* افکت shadow با رنگ قرمز */
    padding: 10px; /* فاصله داخلی دکمه */
}

</string>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdtAge">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>190</y>
     <width>268</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lnEdtAge{
    background-color:  #ffb4b6; /* رنگ پس‌زمینه مشکی */
    color: #3f0002; /* رنگ متن سفید */
	opacity: 0.5;
    border: 2px solid #ffb4b6; /* حاشیه با رنگ قرمز */
    border-radius: 12px; /* گرد کردن حاشیه */
	border-style: inset;
    padding: 3px 10px; /* فاصله داخلی دکمه */
    font-size: 20px; /* اندازه‌ی متن */
	font-family:'Cascadia Mono' ;
	font-weight: bold;
    text-align: center; /* مرکز‌چین کردن متن */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNatCode">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblNatCode {
    color: white; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
    font-size: 27px; /* اندازه‌ی متن (مثلاً 20 پیکسل) */
    border-bottom: 2px solid red; /* حاشیه زیر متن با رنگ قرمز */
    box-shadow: 20 20 30px red; /* افکت shadow با رنگ قرمز */
    padding: 10px; /* فاصله داخلی دکمه */
}

</string>
   </property>
   <property name="text">
    <string>National code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblUser {
    color: white; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
    font-size: 27px; /* اندازه‌ی متن (مثلاً 20 پیکسل) */
    border-bottom: 2px solid red; /* حاشیه زیر متن با رنگ قرمز */
    box-shadow: 20 20 30px red; /* افکت shadow با رنگ قرمز */
    padding: 10px; /* فاصله داخلی دکمه */
}

</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPass">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>275</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblPass {
    color: white; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
    font-size: 27px; /* اندازه‌ی متن (مثلاً 20 پیکسل) */
    border-bottom: 2px solid red; /* حاشیه زیر متن با رنگ قرمز */
    box-shadow: 20 20 30px red; /* افکت shadow با رنگ قرمز */
    padding: 10px; /* فاصله داخلی دکمه */
}

</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdtNatCode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lnEdtNatCode{
    background-color:  #ffb4b6; /* رنگ پس‌زمینه مشکی */
    color: #3f0002; /* رنگ متن سفید */
	opacity: 0.5;
    border: 2px solid #ffb4b6; /* حاشیه با رنگ قرمز */
    border-radius: 12px; /* گرد کردن حاشیه */
	border-style: inset;
    padding: 3px 10px; /* فاصله داخلی دکمه */
    font-size: 20px; /* اندازه‌ی متن */
	font-family:'Cascadia Mono' ;
	font-weight: bold;
    text-align: center; /* مرکز‌چین کردن متن */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdtUser">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lnEdtUser{
    background-color:  #ffb4b6; /* رنگ پس‌زمینه مشکی */
    color: #3f0002; /* رنگ متن سفید */
	opacity: 0.5;
    border: 2px solid #ffb4b6; /* حاشیه با رنگ قرمز */
    border-radius: 12px; /* گرد کردن حاشیه */
	border-style: inset;
    padding: 3px 10px; /* فاصله داخلی دکمه */
    font-size: 20px; /* اندازه‌ی متن */
	font-family:'Cascadia Mono' ;
	font-weight: bold;
    text-align: center; /* مرکز‌چین کردن متن */
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdtPass">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>340</y>
     <width>268</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lnEdtPass{
    background-color:  #ffb4b6; /* رنگ پس‌زمینه مشکی */
    color: #3f0002; /* رنگ متن سفید */
	opacity: 0.5;
    border: 2px solid #ffb4b6; /* حاشیه با رنگ قرمز */
    border-radius: 12px; /* گرد کردن حاشیه */
	border-style: inset;
    padding: 3px 10px; /* فاصله داخلی دکمه */
    font-size: 20px; /* اندازه‌ی متن */
	font-family:'Cascadia Mono' ;
	font-weight: bold;
    text-align: center; /* مرکز‌چین کردن متن */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblWrongName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>234</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblWrongName {
    color: #ffa800; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
	font-size: 12px;
}
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSignup">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>444</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnSignup{
    background-color:  #d10000; /* رنگ پس‌زمینه مشکی */
    color: white; /* رنگ متن سفید */
    border: 3px solid #690003; /* حاشیه با رنگ قرمز */
    border-radius: 12px; /* گرد کردن حاشیه */
    padding: 3px 10px; /* فاصله داخلی دکمه */
    font-size: 30px; /* اندازه‌ی متن */
	font-weight: bold;
    text-align: center; /* مرکز‌چین کردن متن */
    box-shadow: 0 0 10px red; /* افکت glow قرمز */
	border-style: inset black;
}
#btnSignup:hover {
    background-color: #990000; /* تغییر رنگ پس‌زمینه به قرمز */
    color: white; /* تغییر رنگ متن به مشکی */
    box-shadow: 0 0 20px red; /* تغییر افکت glow */
}</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblWrongFamily">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>232</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblWrongFamily {
    color: #ffa800; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
	font-size: 12px;
}
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblWrongAge">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>234</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblWrongAge {
    color: #ffa800; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
	font-size: 12px;
}
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblWrongNatCode">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>384</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblWrongNatCode {
    color: #ffa800; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
	font-size: 12px;
}
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblWrongUsername">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>384</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblWrongUsername {
    color: #ffa800; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
	font-size: 12px;
}
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblWrongPassword">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>384</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lblWrongPassword {
    color: #ffa800; /* رنگ متن سفید */
    font-weight: bold; /* متن بولد */
	font-size: 12px;
}
</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
